Documentation/contributing/development/contributing_guide.rst

```diff
@@ -238,78 +244,81 @@ Getting a pull request merged
    #. Address any feedback received from the reviewers
    #. You can push individual commits to address feedback and then rebase your
       branch at the end before merging.
+   #. Once you have addressed the feedback, re-request a review from the
+      reviewers that provided feedback. This ensures that the reviewers are
```

Comment:
Maybe:

```suggestion
   #. Once you have addressed the feedback, re-request a review from the
      reviewers that provided feedback by clicking on the button next to their name, in the list of reviewers. This ensures that the reviewers are
```

Because I'm not sure everyone is familiar with that button.